trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.controller.js

```diff
@@ -1,15 +1,26 @@
 const { format, fromUnixTime, getUnixTime } = require('date-fns');
 const { AMENDMENT_STATUS } = require('@ukef/dtfs2-common');
-const { isTfmFacilityEndDateFeatureFlagEnabled } = require('@ukef/dtfs2-common');
+const { HttpStatusCode } = require('axios');
 const api = require('../../../api');
 const { coverEndDateValidation } = require('./validation/amendCoverEndDateDate.validate');
+const { isFacilityEndDateEnabledForTfmFacility } = require('../../helpers/isFacilityEndDateEnabledForTfmFacility');
+
+const getNextPage = (apiResponseStatus, changeFacilityValue, showFacilityEndDatePage, baseUrl) => {
```

Comment:
Can some documentation be made for this function for what it does and the types for all the params etc please

Reply:
I don't feel like that is needed here as the function is just used internally within the file - I just pulled it out to make the returning next page a bit clearer to read.

Reply:
Added some docs, fwiw I think it's self documenting (I managed to write the docs by reading the function name and input parameters, so we're doubling up on maintainence, but aware this is the general pattern)